docs/source/api/filters.rst

```diff
@@ -0,0 +1,54 @@
+Filters
+=======
```

Comment:
This should probably have a little intro paragraph explaining that these are for use in templates.